composer.json

```diff
@@ -65,8 +65,7 @@
     "monolog/monolog": "1.23.*",
     "newerton/jquery-mousewheel": "dev-master",
     "pamelafox/lscache": "1.0.5",
-    "malihu/malihu-custom-scrollbar-plugin": "3.1.5",
-    "grimmlink/qtip2": "3.0.3"
+    "malihu/malihu-custom-scrollbar-plugin": "3.1.5"
```

Comment:
@osma ah! I think it was the GitHub UI's diff, showing the entry as removed because I removed the comma? Still good to check, thanks!

Reply:
Ah, you are right. I misread the diff. Sorry for the noise!